resources/bin/ballerina

```diff
@@ -39,8 +39,8 @@ fi
 if [ "$RUN_COMMAND" == "true" ]
 then
     JAVA_COMMAND=java
-    if test -d "$CURRENT_PATH/../dependencies/jdk8u202-b08-jre"; then
-      JAVA_COMMAND="$CURRENT_PATH/../dependencies/jdk8u202-b08-jre/bin/java"
+    if test -d "$CURRENT_PATH/../dependencies/jdk-11.0.8+10-jre"; then
+      JAVA_COMMAND="$CURRENT_PATH/../dependencies/jdk-11.0.8+10-jre/bin/java"
     elif [[ `$JAVA_COMMAND -version 2>&1 | grep "1.8."` = "" ]]; then
       echo "Compatible JRE 8 not found. Please follow the instructions in <BALLERINA_HOME>/INSTALL.txt to install and setup Ballerina."
```

Comment:
Don't we need to modify this error message?

Reply:
error message + checking condition should be updated I guess